README.md

```diff
@@ -1 +1,48 @@
-# FairClassifier
\ No newline at end of file
+# FairClassifier
+
+**Machine Learning** models are extensively used for the purpose of determining access to services such as eligibility for loan, insurance etc. Despite gains in efficiency of these models, unintentional discrimination flaw of the model has not been fully addressed. 
+
+The Fair Classifier is an open source package that evaluate the fairness of a Neural Network. The goal is to build a command-line playground where user can check the fairness of the model by removing different sensitive attribute from the training data.
+ 
+This python package includes a set of metrics to test models for biasness and use adversarial network to help mitigate that biasness of the training data set.
+
+
+### Fairness metrics implemented
+> The fairness contraint P% rule is based on this paper [Link.](https://arxiv.org/pdf/1507.05259.pdf)
```

Comment:
Insignificant comment, I would do `P% rule is based on [this paper](https://....).` but that's just stylistic

Reply:
done